### PR TITLE
Power saving mode

### PR DESCRIPTION
**Pull request by alexeysirenko**

### CODE REVIEW

File: server.py [hunk 1]
```diff
@@ -265,15 +266,24 @@ def healthcheck():
     cache_updated_ms_ago = now - app.cache_last_updated_at if app.cache_last_updated_at > 0 else None
     settings_file_exists = _kraken_settings_file_exists()
     doa_file_exists = _kraken_doa_file_exists()
-    status_ok = doa_file_exists and doa_updated_ms_ago < 1000 and settings_file_exists
-    return {
+    doa_ok = doa_updated_ms_ago < 1000 and doa_file_exists if doa_updated_ms_ago else False
+    kraken_service_running = is_kraken_service_running()
```
Comment (dubrsl) [inline server.py:270]: it will not work in docker. You can't manage kraken service from other instance.

Comment (alexeysirenko) [reply server.py:283]: I know

Comment (dubrsl) [reply server.py:283]: Ok, So, what is solution?

Comment (alexeysirenko) [reply server.py:283]: Does it block dockerization completely or just some healthchecks do not pass?

Comment (dubrsl) [reply server.py:283]: Some /healthcheck fields will be null. But it's possible update docker configuration to access to usb and gpiomem devices and to /sys/class/thermal fs.
But with systemd is more difficult.  Of course, we can install a fake service that will manage the Docker instance via a mounted socket. But installation the systemd in instance is not good idea.

**Pull request merged**